sdk/java-sdk/src/main/java/kalix/javasdk/Kalix.java

```diff
@@ -337,7 +337,7 @@ public <D extends ReplicatedData, E extends ReplicatedEntity<D>> Kalix register(
    *
    * @return This stateful service builder.
    */
-  public <S, E extends ValueEntity<S>> Kalix register(ValueEntityProvider<S, E> provider) {
+  public Kalix register(ValueEntityProvider<?, ?> provider) {
```

Comment:
Leftover. 
I got some issue with inference and reflection, but I think I can revert this change now.